site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: "Install Agones using Helm"
-linkTitle: "Install with Helm"
-weight: 4
+linkTitle: "Helm"
+weight: 20
 description: >
-  This chart install the Agones application and defines deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+  Installs the Agones on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
 
 ---
 
```

Comment:
In the prerequisites section below, can you copy the bullet "If you are creating and managing your own Kubernetes cluster, the MutatingAdmissionWebhook, and ValidatingAdmissionWebhook admission controllers are required. We also recommend following the recommended set of admission controllers." (with links) from 
site/content/en/docs/Installation/_index.md instead of having something that looks similar?

Reply:
I made a link of "Supported Kubernetes Cluster" instead - we don't list K8s pre-reqs in the index.yaml installation, and having the same info in two places feels reduntant. PTAL.

Reply:
Perfect.